admin/templates/configuration/etc/init.d/mysql-proxy

```diff
@@ -70,7 +71,7 @@ if [ "$USER" = "root" ]; then
     proxy_user_option="--user=$proxy_user"
 fi
 
-cmd="DEBUG=${LUA_DEBUG_LEVEL} QSERV_RPC_PORT=${QSERV_RPC_PORT} ${DAEMON} \
+cmd="DEBUG=${LUA_DEBUG_LEVEL} QSERV_RPC_PORT=${QSERV_RPC_PORT} QSERV_CONFIG=${QSERV_CONFIG} ${DAEMON} \
```

Comment:
Passing configuration trough environment variable isn't nice, we should open a ticket to solve this, isn't it?